### PR TITLE
add riot.js plugin

### PR DESCRIPTION
**Pull request by ariesjia**

### CODE REVIEW

File: docs/ecosystem-riot.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+id: ecosystem-riot
+title: single-spa-riot
+sidebar_label: riot
+---
+
+single-spa-riot is a helper library that helps implement [single-spa registered application](single-spa-config.md#registering-applications) [lifecycle functions](building-applications.md#registered-application-lifecycle) (bootstrap, mount and unmount) for for use with [riot](https://riot.js.org/). Check out the [single-spa-svelte github](https://github.com/ariesjia/single-spa-riot).
```
Comment (frehner) [inline docs/ecosystem-riot.md]: Single-spa-svelte github? :D

**Review by frehner — COMMENTED** [review]
One small typo :)

File: docs/ecosystem-riot.md (hunk 1 above)
Comment (ariesjia) [reply docs/ecosystem-riot.md]: I copied from the svelte article.

**Comment by frehner** [comment]
Thank you for your contribution! 

Feel free to join our slack channel as well if you're not there already.

**Pull request merged**